CONTRIBUTING.md

```diff
@@ -56,15 +56,15 @@ if you don't.
 
 For more details, see <https://docs.astral.sh/ruff/>.
 
-## Type-checking
+## Type-checking and Testing
 
-To validate that the annotations are correct, we use two type-checkers:
-[basedpyright] and [mypy]. Both are included as development
-dependency, and therefore automatically installed with `uv sync`.
+To validate that the annotations are correct and run tests, we use several tools:
+[basedpyright], [mypy], and [pytest]. All are included as development
+dependencies, and therefore automatically installed with `uv sync`.
```

Comment:
I don't think we should combine the static type-checking and runtime testing topics like this. Pytest probably deserves its own section, where we can also explain where the tests can be found, what their purpose is, and how to contribute to them.